armtemplates/linux-webapp-template.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
should we move this file to linux-webapp folder to keep folder consistency with windows-webapp